java/ql/test/query-tests/security/CWE-730/RegexInjectionTest.ql

```diff
@@ -0,0 +1,21 @@
+import java
+import TestUtilities.InlineExpectationsTest
+import semmle.code.java.security.regexp.RegexInjectionQuery
+
+//import semmle.code.java.security.regexp.PolynomialReDoSQuery
```

Comment:
This can be removed.

```suggestion
```

Reply:
Updated in https://github.com/github/codeql/pull/11070/commits/f64572d432251f62d05755816481e476563d8e1a.